styleguide/_themes/derek/scss/patterns/grid/keyOffers.scss

```diff
@@ -1,35 +1,69 @@
-.offerBox {
+.keyoffers-col-full {
+  @include make-sm-column(12);
+  @include flex-box;
+}
+
+.keyoffers-col-half {
+  @include make-sm-column(6);
+  @include flex-box;
+}
+
+.keyoffers-col-third {
+  @include make-xs-column(12);
+  @include make-md-column(4);
+  @include flex-box;
+}
+
+.keyoffersWidget {
   background: $white;
-  background-clip: padding-box;
-  margin: 10px;
+  display: block;
+  margin-bottom: 10px;
   padding: 25px;
+  text-decoration: none;
+}
 
-  p {
-    line-height: 1.7em;
-    padding-bottom: 10px;
-  }
+.keyoffersWidget-body {
+  line-height: 1.7em;
+  padding-bottom: 10px;
+}
 
-  .bottom {
-    i {
-      bottom: 0;
-      float: right;
-      height: 50px;
-      position: absolute;
-      right: 15px;
-      width: 50px;
-    }
-
-    p {
-      bottom: 0;
-      float: left;
-      left: 25px;
-      padding-bottom: 0;
-      position: absolute;
-    }
-  }
```

Comment:
This is not the only place "bottom" is used. There's things on www utilizing this I think. We may need a very solid QE sweep around this one.